### PR TITLE
Added MinimumElementSnippet

### PR DESCRIPTION
**Pull request by varada1110**

### CODE REVIEW

**Review by iluwatar — CHANGES_REQUESTED** [review]
Looks good, but the build is failing due to Checkstyle errors

File: README.md [hunk 1]
```diff
@@ -251,6 +252,20 @@ Inspired by [30 seconds of code](https://github.com/Chalarangelo/30-seconds-of-c
...
+```java
+  public static int findMin(int[] array) {
+        int min = array[0];
+        for (int i = 1; i < array.length; i++) {
+        if (min > array[i]) {
+        min = array[i];
+        }
+        }
+        return min;
+        }
```
Comment (iluwatar) [inline README.md:266]: The intendation isn't right here

File: build.gradle [hunk 2]
```diff
@@ -61,6 +61,8 @@ dependencies {
     testImplementation 'org.hamcrest:hamcrest-library:1.3'
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.2'
     testImplementation 'org.junit.jupiter:junit-jupiter-params:5.6.2'
+    testImplementation 'junit:junit:4.13.1'
+    testImplementation 'junit:junit:4.13.1'
```
Comment (iluwatar) [inline build.gradle:65]: The dependency should not appear twice

**Review by iluwatar — CHANGES_REQUESTED** [review]
Two minor comments, then we can merge

**Review by iluwatar — CHANGES_REQUESTED** [review]
Looks good otherwise, but the build is still failing. I can't merge until the Checkstyle issues have been fixed.

**Review by PraveenNanda124 — COMMENTED** [review]
Good work

**Pull request closed without merging**